src/lib/__tests__/server-utilities.test.js

```diff
@@ -0,0 +1,18 @@
+import { md5Hash } from 'lib/server-utilities';
+jest.unmock('lib/server-utilities');
+
+describe('md5Hash', () => {
+  const adminClientHash = md5Hash('./static/build/editor-client.js');
+  it('returns a string', () => {
+    expect(typeof adminClientHash).toBe('string');
+  });
+
+  it('hashes consistently', () => {
+    const secondAdminClientHash = md5Hash('./static/build/editor-client.js');
+    expect(adminClientHash).toBe(secondAdminClientHash);
+  });
```

Comment:
Is there a chance of an inconsistent hash? Not that you should remove the test _per se_, but I can't imagine this happening.